index.html

```diff
@@ -58,9 +69,25 @@
     <script type="text/javascript" src="public/d3.js"></script>
     <script type="text/javascript" src="public/bitemp_graph.js"></script>
     <script type="text/javascript" src="public/bitemp_graph_helper.js"></script>
+    <script type="text/javascript" src="public/input_text_load_data.js"></script>
 
     <div id="bar-chart-large"></div>
-	
+  
+    <script type="text/javascript">
+      $("input[name='insert']").click(function(){
```

Comment:
That was in index.html before we got it. I am not too sure what it is doing either. Should I remove it?